be/src/main/java/yeonba/be/arrow/service/ArrowService.java

```diff
@@ -50,4 +55,39 @@ public UserArrowsResponse getUserArrows(long userId) {
 
     return new UserArrowsResponse(user.getArrow());
   }
+
+  /*
+  화살 보내기 비즈니스 로직은 다음 과정을 거친다.
+  1. 자기 자신에게 화살을 보내는 상황 검증
+  2. 화살을 보낸 사용자에게 또 보내는 상황 검증
+  3. 화살 내역 저장
+  4. 보내는 사용자 화살 감소, 화살이 부족할 경우 예외 발생
+  5. 받는 사용자 화살 증가
+
+ */
```

Comment:
사소한 부분이지만 다음 PR부터는 신경써주세요.

```suggestion
  5. 받는 사용자 화살 증가
 */
```